docs/application/dotnet/get-started/wearable/first-xaml-app.md

```diff
@@ -126,6 +130,30 @@ If you face any issues during deployment, it is recommended to manually install
 ```bash
 $ sdb install <path-to-package>/org.tizen.example.TizenWearableXamlApp1-1.0.0.tpk
 ```
+## Validate Your Application Changes Using XAML Hot Reload
+
+To test your UI code changes in your application, you would have to build and deploy your application. With the introduction of the XAML Hot Reload feature, you can now immediately see the UI edits on the running emulator without having to rebuild or redeploy your application. This feature helps you save time and validate your changes easily.
+
+To use the Hot Reload feature, you must enable it in your Visual Studio IDE. To enable:
+
+1. In the Visual Studio menu, select **Tools** > **Options**.
+2. In the **Options** window that appears, select **Tizen** > **Tools**.
+3. Select **Enable XAML Hot Reload**.
+
+    ![Enable Hot Relaod](media/vs_enable_hot_reload.PNG)
+
+### Using XAML Hot Reload 
+
+You can launch the instance in debugging mode and run the application by clicking the emulator instance in the Visual Studio toolbar.
+
+![Deploy your package](media/vs_emulator_launch_xaml.png)
+
+After you make UI changes in your application code, you do not need to rebuild or redeploy the application.
+
+You will see that your changes are immediately reflected on your emulator while you are in debugging mode.
```

Comment:
@surbhig12 
One question about the tone of the sentences.

In the updated parts, I can see the tone like "you will see", "you can now immediately see", "you would have to".
It gives me more familiar impression than other parts,
as at the other parts, "~ appears/changes", "you must" tone is used.

So, to me, this parts gives me the impression of different tone compared to left.
Isn't it better to use unified tone in one page? Would you please give me your opinion to my question?

Reply:
@mijin85cho,
In the newly added part the tone has been kept similar to the tone in the current page. However, you may feel that in certain sentences the tone is different. For example, 
for the sentence: To test your UI code changes in your application, you would have to build and deploy your application.

If we change the sentence to "To test your UI code changes in your application, you must build and deploy your application. With the introduction of the XAML...", then the flow and meaning changes.

Therefore, to avoid changing the meaning, we can keep it as is.
Let me know your views.

Reply:
> If we change the sentence to "To test your UI code changes in your application, you must build and deploy your application. With the introduction of the XAML...", then the flow and meaning changes.

Would you please let me the changes in detail? What flow and meaning are changed?